package.json

```diff
@@ -351,12 +351,13 @@
     "react-router-last-location": "^2.0.1",
     "react-scroll": "1.7.10",
     "react-tabs": "^3.1.1",
-    "react-transition-group": "1",
+    "react-transition-group": "^1.2.1",
     "recompose": "^0.30.0",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
     "regenerator-runtime": "^0.13.7",
     "reselect": "^2.5.4",
+    "styled-components": "^5.2.3",
```

Comment:
Is there a reason we're not using the design system classes for these styles?

Reply:
yes: https://github.com/department-of-veterans-affairs/vets-website/pull/16759#discussion_r612586483

Reply:
also, related to the design system.  it was applying this rule to all buttons inside an accordion smh:

// .usa-accordion > ul button, .usa-accordion-bordered > ul button

which would add a background image and a bunch of padding to any button inside an accordion.

so i had to use !important due to the design system. 

do you have any suggestions here? 

const UnStyledButtonInAccordion = styled.button`
  padding: 0px !important;
  background-color: transparent !important;
  background-image: none !important;
  width: auto !important;
`;

Reply:
even ` className="usa-button-unstyled"` did not do it.

Reply:
This is a pretty significant architectural departure, I think we should get @cvalarida eyes on this, it feels like an awful lot of weight for my much gain to me

Reply:
i think these requirements exposed some limitations in our architecture (namely dynamic styling inside a drupal template based on a react variable). what about it specifically is weighty?

feel free to take a look at the docs re: any concerns about adding to the bundle size / use cases, etc: https://styled-components.com/

Reply:
Hard to debate some of these, but I would request that you take a look at the design system's utility classes so we can at least replace the ones in there that we have w/o duplication. Probably not a big deal at a singular level, but if every application followed the pattern of adding all their CSS like this we'd continue to bloat the package and system unnecessarily doing some real cumulative damage to performance.

Here are the utilities: https://design.va.gov/utilities

Reply:
thank you! I always try to default to using the css utilities where/when i can.